src/modules/ipfix/CollectorCfg.h

```diff
@@ -142,10 +148,8 @@ class CollectorCfg
 			(mtu == other->mtu) &&
 			(peerFqdns == other->peerFqdns) &&
 			(buffer == other->buffer) &&
-			(authorizedHosts.size() == other->authorizedHosts.size())) {
-			for (uint16_t i = 0; i < authorizedHosts.size(); i++)
-				if (authorizedHosts[i] != other->authorizedHosts[i])
-					return false;	
+			(authorizedHosts.size() == other->authorizedHosts.size()) &&
```

Comment:
I think this check can be removed since it is implicitly covered by the following `==`. And it would surprise me if the `==` implementation did not check the length first as an optimization.